src/exec/mod.rs

```diff
@@ -31,13 +38,53 @@ use once_cell::sync::Lazy;
 /// environment, and then actually executes the tenant's workload.
 ///
 /// Basically, this is a generic view of exec_wasmtime.
-pub trait Workldr: Sync + Send {
+pub trait Exec: Sync + Send {
     /// The name of the Workldr
     fn name(&self) -> &'static str;
 
     /// The builtin Workldr binary (e.g. exec_wasmtime)
     fn exec(&self) -> &'static [u8];
+
+    fn with_backend(&self, backend: &dyn Backend) -> bool;
```

Comment:
Should we add a comment documenting what it does?

Reply:
done